mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -782,6 +782,11 @@ def LLVM_CallOp : LLVM_MemAccessOpBase<"call",
     /// Returns the callee function type.
     LLVMFunctionType getCalleeFunctionType();
   }];
+  let extraClassDefinition = [{
+    std::optional<::mlir::StringRef> $cppClass::getUserSymbol() {
```

Comment:
I suggest to add this next to https://github.com/trailofbits/instafix-llvm/blob/f21b247d13cda3a9f844078aeacaad5bd17d3905/mlir/lib/Dialect/LLVMIR/IR/LLVMDialect.cpp#L1207 instead of `extraClassDefinition` as rest of the class is implemented there anyway